sys/include/net/netopt.h

```diff
@@ -269,6 +269,54 @@ typedef enum {
      */
     NETOPT_LAST_ED_LEVEL,
 
+    /** @brief   Radio modulation bandwidth
+     *
+     * Depends on the type of radio used and the regional regulatory and is
+     * used with LoRa/FSK modems.
+     *
+     * Actual implementations support 125kHz, 250kHz and 500kHz.
+     *
+     * @see drivers_sx127x
+     */
+    NETOPT_BANDWIDTH,
+
+    /** @brief   Radio spreading factor
+     *
+     * For LoRa/FSK modems, allowed values are in the range 7..12.
+     */
+    NETOPT_SPREADING_FACTOR,
+
+    /** @brief   Radio coding rate
+     *
+     * For LoRa/FSK modems, allowed values are in the range 5..8.
+     */
+    NETOPT_CODING_RATE,
+
+    /** @brief   Semtech module modem type.
+     *
+     * Allowed values are LoRa and FSK.
+     *
+     * FSK is not supported for the moment.
+     */
+    NETOPT_MODEM_TYPE,
+
+    /**
+     * @brief LoRA specific options.
+     */
+    NETOPT_LORA_SINGLE_RECEIVE,      /**< When enabled, RX is turned off upon reception of a packet */
```

Comment:
Me neither, but they could, so there is no reason (and no harm in doing not so) to make it LoRA specific.